brevproduksjon/src/main/java/no/nav/foreldrepenger/fpformidling/brevproduksjon/bestiller/DokgenBrevproduksjonTjeneste.java

```diff
@@ -143,11 +146,11 @@ private void distribuerBrevOgLagHistorikk(DokumentHendelse dokumentHendelse, Dok
                         innsynMedVedlegg,
                         dokumentHendelse.getBehandlingUuid(),
                         saksnummer,
-                        dokumentHendelse.getBestillingUuid()));
+                        UUID.randomUUID())); //ToDo ANJA fjerne når vi ikke lenger krever Uuid på historikkinnslag
 
         taskGruppe.addNesteSekvensiell(
                 opprettPubliserHistorikkTask(dokumentHendelse.getBehandlingUuid(),
-                        dokumentHendelse.getBestillingUuid(),
+                        UUID.randomUUID(), //ToDo ANJA fjerne når vi ikke lenger krever Uuid på historikkinnslag
```

Comment:
Den må nok bli satt til dokumentHendelse.getBestillingUuid() siden vi vil at JournalpostId kobles mot riktig BehanadlingDokumentBestillt dokument. 

Den må da utvides til å takle flere kall med samme bestilling_uuid.

Reply:
Skulle til å spørre @mrsladek om dette men du kom meg i forkjøpet.
Hva tror du er best: å sende flere historikkrequests til fpsak eller å utvide kontrakten til å bli en liste av journalpostid/dokumentId og kalle en gang?

Reply:
Foreslår at fpsak bare lager historikkinnslag for det andre dokumentet og lar bestillingstabellen være i fred hvis det allerede er kvittert. Evt bare gjør en update.
Det er to brev og journalpostId'er men samme hvilken vi lagrer - så lenge det finnes en. Ser ikke behovet for å lagre begge - det kan man finne fra SAF og se i dokumentlisten

Reply:
Ja det er nok det som skjer i dag - vi bare oppdaterer journalpostId - som du sier innhold i brevet er likt uansett så det bør ikke spille en stor rolle. Det lagres to historikkinnslag også (etter jeg fjerner den rare historikkUuiden https://github.com/navikt/fp-sak/pull/3994).